Resources/ServerInfo/Guidebook/Slang.xml

```diff
@@ -0,0 +1,31 @@
+<Document>
+  # Slang Terms
+  Space Station 14 players tend to use a lot of slang in their language to cut down on time spent typing.
+
+  Common IC (In Character) terms are as follows:
+  - [color=#a4885c]Antag[/color]: Shorthand for antagonist; the various people,creatures, and other abominations who pose a threat to the station and crew.
```

Comment:
Just use NanoGold instead of hardcoding a hex value.
if doc parsing doesn't support it, that needs to be added

Reply:
nevermind, apparently rich text doesn't currently support stylesheets